Automattic/wp-calypso
# Enable Simple Sites on Jetpack Cloud

Related to https://github.com/Automattic/dotcom-forge/issues/6509

## Proposed Changes

Enable Simple Sites on Jetpack Cloud (dev environment) and update Upsell screens to show WPcom plans instead of Jetpack Products.

#### Upsell sections

- [x] Activity Log: Show Upsell with `Starter` plan when the site is Free.
- [x] Backup: Show Upsell with `Creator` plan.
- [x] Backup: Show `Activate` when the plan is >= Creator, and it's not Atomic.
- [ ] Search: Show Upsell with `Creator` plan. (won't implement in this PR)
- [ ] Social: Show Upsell with `Creator` plan. (won't implement in this PR)
- [x] Subscriptions > WordAds: Show Upsell with `Explorer` plan (already existed) 

| Before | After |
| --- | --- |
| ![image](https://github.com/Automattic/wp-calypso/assets/402286/626d621b-fd79-4ce7-a117-33dfacd79c99) | ![image](https://github.com/Automattic/wp-calypso/assets/402286/e070f938-4df5-4ea8-ba82-aef34b411398)|
| ![image](https://github.com/Automattic/wp-calypso/assets/402286/197d1a66-2c1d-44d7-872c-0f927ae9525d) |![image](https://github.com/Automattic/wp-calypso/assets/402286/59d12fd2-a840-4a65-b71f-c9de0e3b56e0) ![image](https://github.com/Automattic/wp-calypso/assets/402286/d3b14412-1e50-45e5-841c-61afbd06d933) ![image](https://github.com/Automattic/wp-calypso/assets/402286/4f82dc49-d09a-49ae-b8c3-9c222f61e3e0) |
| ![image](https://github.com/Automattic/wp-calypso/assets/402286/16974cd6-f9b3-4c33-912c-a89dbfb008c5) |![image](https://github.com/Automattic/wp-calypso/assets/402286/9cc2ec01-2bd7-4ca7-986a-4b013d7df085)|
| ![image](https://github.com/Automattic/wp-calypso/assets/402286/d48e3bca-bf73-4923-8a0f-e10c7dffadb3) | TBA |
| ![image](https://github.com/Automattic/wp-calypso/assets/402286/5b452b5a-84d0-45c4-ac35-24a129a4a859) | ![image](https://github.com/Automattic/wp-calypso/assets/402286/b8479312-e299-4571-b2dd-3be139bef136)|


## Testing Instructions

Use the Jetpack Cloud live link below, or check out the branch and set up your local env for testing:

* Run `yarn start-jetpack-cloud`
* Go to `http://jetpack.cloud.localhost:3000/`

Regression testing (this is most important for this PR):

Feel free to use the live link to do the regression testing against production. Or remove the "wpcom" `site_filter` in `jetpack-cloud-development.json`.

- [ ] /backups, Backup and Search and Scan upsells (including when Atomic transfer is needed), /activity-log and A4A dashboard activity log (/sites/overview/{sites}/jetpack-activity), and /earn still display correctly.
- [ ] Jetpack Cloud screens aren't broken for Atomic and Jetpack sites.
- [ ] We show the correct screen when the user has no sites (or no "visible" sites).
